CourseManager.xcodeproj/xcuserdata/summercrow.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -36,5 +36,37 @@
             landmarkType = "24">
          </BreakpointContent>
       </BreakpointProxy>
+      <BreakpointProxy
```

Comment:
@sghourab add please these files to `.gitignore`

Reply:
How would I find these files?

Reply:
Is this how you would do it? I read online that i need to temporarily delete file, then commit and re add it. If so, I don't know how to access it from xcode. Should delete it from finder instead?

<img width="829" alt="Screenshot 2022-12-20 at 9 56 29 AM" src="https://user-images.githubusercontent.com/44596576/208697577-9fbabb14-ec1d-442e-8c0a-f2a457e70e0a.png">